vocabularies/earth-science-data-category.ttl

```diff
@@ -23,545 +23,600 @@
     skos:exactMatch <http://purl.org/au-research/vocabulary/anzsrc-for/2008/0403> ;
     skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
     skos:notation "0403" ;
     skos:prefLabel "Geology"@en .
 
-:airborne-spectral a skos:Concept ;
-    skos:broader :spectral-imaging ;
+esdc:airborne-spectral a skos:Concept ;
+    skos:broader esdc:spectral-imaging ;
     skos:definition "Remotely sensed spectral imaging obtained from an airborne platform."@en ;
     skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
     skos:prefLabel "Airborne Spectral"@en .
 	
-:core-spectral a skos:Concept ;
-    skos:broader :spectral-imaging ;
+esdc:core-spectral a skos:Concept ;
+    skos:broader esdc:spectral-imaging ;
     skos:definition "Spectral imaging (typically hyperspectral) of rock samples such as core."@en ;
     skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
     skos:prefLabel "Core Spectral"@en .
 
-:ecology a skos:Concept ;
-    skos:broader :biological-sciences ;
+esdc:ecology a skos:Concept ;
+    skos:broader esdc:biological-sciences ;
     skos:definition "A branch of biology concerning interactions among organisms and their biophysical environment, which includes both biotic and abiotic components."@en ;
     skos:exactMatch <http://purl.org/au-research/vocabulary/anzsrc-for/2008/0602> ;
     skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
     skos:notation "0602" ;
     skos:prefLabel "Ecology"@en .
 
-:electrical-geophysics a skos:Concept ;
-    skos:broader :electrical-and-electromagnetic-methods-in-geophysics ;
+esdc:electrical-geophysics a skos:Concept ;
+    skos:broader esdc:electrical-and-electromagnetic-methods-in-geophysics ;
     skos:definition "There are a variety of different electrical methods that used to image different electrical properties of the subsurface. The Resistivity technique is used to determine the resistivity of subsurface materials, the Induced Polarization (IP) technique is used to measure the decay of voltage in the ground, and the Self Potential (SP) technique looks for natural voltage sources underground. Source: Dept. Energy and Mining, Gov. of S.A."@en ;
     skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
     skos:prefLabel "Electrical Geophysics"@en .
 
-:geomechanics a skos:Concept ;
-    skos:broader :geology ;
+esdc:electromagnetic-geophysics a skos:Concept ;
+    skos:broader esdc:electrical-and-electromagnetic-methods-in-geophysics ;
+    skos:definition "An electromagnetic survey is based on the response of the ground to the propagation of electromagnetic fields composed of an alternating electric intensity and magnetizing force. source: <https://www.sciencedirect.com/topics/earth-and-planetary-sciences/electromagnetic-survey> "@en ;

```

Comment:
Please remove the < > from the URL in this definition.

Reply:
Done!